### ISSUE TITLE
Find View hangs applications with large graphs open 

### ISSUE THREAD

**Issue by antares1470** [constellation-app/constellation#852]
### Prerequisites

* [ ] Put an X between the brackets on this line if you have done all of the following:
    * Running the latest version of Constellation
	* Attached the ***Support Package*** via `Help` > `Support Package`
    * Checked the FAQs: https://github.com/constellation-app/constellation/wiki/FAQ
    * Checked that your issue isn't already filed: https://github.com/constellation-app/constellation/issues
    * Checked that there is not already a module that provides the described functionality: https://github.com/constellation-app/constellation/wiki/Catalogue-of-Repositories

### Description

When several large graphs are open in Constellation, opening find view causes the application to hang and lag. The issue may be to do with updating the find view state.

### Steps to Reproduce

1. Open 4 sphere graphs with 50k nodes each
2. Open Find View
3. Try using Constellation

**Expected behaviour:** Find View doesn't cause application to hang

**Actual behaviour:** Find view causes application to hang with several large graphs open

**Reproduces how often:** 100%

### Additional Information

Was found while testing #328

**Comment by serpens24** [comment]
I beleive arcturus mentioned that the issue occurs when switching between open graphs weith find view (and other views) all open.

**Comment by Nova-2119** [comment]
So I wasn't able to get the application to hang but I did consistently get an error. I've fixed the source of the error and I'm hoping that will fix the hanging issue described.

It feels a bit like putting a band-aid on when a proper bandage is required though. Its still acting a bit odd, for instance when you switch graphs the view might not appear, particularly if you are switching to that graph for the first time.
I think ideally it needs re-writing or atleast examining in depth, but I think that about the analytic view aswell so maybe I just don't like views!
